packages/flutter/test/material/search_test.dart

```diff
@@ -994,6 +994,27 @@ void main() {
     expect(selectedResults, <String>['Result']);
   });
 
+  testWidgets('Leading width size is 16', (WidgetTester tester) async {
+    final _TestSearchDelegate delegate = _TestSearchDelegate();
+    final List<String> selectedResults = <String>[];
+    delegate.leadingWidth = 16;
+
+    await tester.pumpWidget(TestHomePage(
+      delegate: delegate,
+      results: selectedResults,
+    ));
+
+    // Open the search page with check leading width smaller than 16.
```

Comment:
Maybe here we can check if the underlying `AppBar` has `leadingWidth` property correctly set to `16`.

Reply:
i added the leading width check. thank you